scale/job/models.py

```diff
@@ -1302,6 +1306,13 @@ def configure_docker_params(self, workspaces, docker_volumes):
         # Configure any Docker parameters needed for workspaces
         configuration.configure_workspace_docker_params(self, workspaces, docker_volumes)
 
+        # Add job environment variable as docker parameters
+        interface = self.get_job_interface().get_dict()
+        for env_var in interface['env_vars']:
+            env_var_name = env_var['name']
+            env_var_value = env_var['value']
+            configuration.add_job_task_docker_params([DockerParam('env', env_var_name + '=' + env_var_value)])
+
```

Comment:
The env var values need to have the same string replacement done to them that is done to the command arguments.